api/models/comments-models.js

```diff
@@ -33,3 +32,20 @@ exports.selectCommentsByArticleId = (article_id) => {
       return rows;
     });
 };
+
+exports.removeCommentById = (comment_id) => {
+  return db
+    .query(
+      `
+    DELETE FROM comments
+    WHERE comment_id = $1
+    RETURNING *;
+  `,
+      [comment_id]
+    )
+    .then(({ rows }) => {
+      if (!rows.length) {
+        return Promise.reject({ status: 404, msg: "comment not found" });
+      }
+    });
+};
```

Comment:
Nice use of `RETURNING *` here to check if the comment was found, and relying on PSQL errors to handle the rest